pkg/controller/clusterdeployment/installconfigvalidation.go

```diff
@@ -11,11 +11,13 @@ import (
 )
 
 const (
-	noAWSPlatformErr     = "install config did not contain an AWS platform"
-	noGCPPlatformErr     = "install config did not contain a GCP platform"
-	noAzurePlatformErr   = "install config did not contain an Azure platform"
-	novSpherePlatformErr = "install config did not contain an vSphere platform"
-	regionMismatchErr    = "install config region does not match cluster deployment region"
+	noAWSPlatformErr        = "install config did not contain an AWS platform"
+	noGCPPlatformErr        = "install config did not contain a GCP platform"
+	noAzurePlatformErr      = "install config did not contain an Azure platform"
+	novSpherePlatformErr    = "install config did not contain an vSphere platform"
+	noNutanixPlatformErr    = "install config did not contain an Nutanix platform"
+	regionMismatchErr       = "install config region does not match cluster deployment region"
+	prismCentralMismatchErr = "install config Prism Central does not match cluster deployment Prism Central"
```

Comment:
Slippery slope here. Are we going to validate *all* of the overlapping fields?

I'm still not happy with the idea of duplicating fields from the install-config.

But I've been sitting here for an hour thinking through the alternatives, and none of them are wonderful enough to warrant nutanix having a totally different paradigm in among all the other providers :(

Reply:
Yup, I'm on board.